BENCHMARKING.md

```diff
@@ -0,0 +1,28 @@
+Benchmarking
+============
+
+We highly encourage the use of benchmarks to measure and improve the performance of Kwaak.
+
+Kwaak is supported by the agent test harness we've developed to compare agent performance across different benchmarks
+and LLM's. You can find it [here](https://github.com/bosun-ai/agent-test-harness).
+
+Specifically to run the SWE-bench using the test harness, run the following command:
+
+```shell
+git clone https://github.com/bosun-ai/agent-test-harness
+cd agent-test-harness
+cargo install amsterdam derrick
+export OPENAI_API_KEY=YOUR_OPENAI_API_KEY
+uv run agent-test-swe
```

Comment:
Does this run the full swe bench? Is there a way to run a selection?

A nice addition would be a quick overview of amsterdam/derrick and the test harness does. Also, how does this run kwaak?

Reply:
@tinco

Reply:
It would run the full swe but it's currently hardcoded to only run a subset.

Reply:
@tinco Do you want to merge this in? Or does it make sense to wait for the trimmed harness you are working on?